configure

```diff
@@ -954,18 +960,73 @@ sub targets_win64 {
   return $arch eq 'x64';
 }
 
+sub system_default_install_dir {
+  if (is_windows()) {
+    return targets_win64() ? $ENV{'ProgramFiles'} : $ENV{'ProgramFiles(x86)'};
+  }
+  return '/usr';
+}
+
+## Detect Google Test
```

Comment:
This is completely clear without the comment (same with "detect RapidJSON" and "Build google test")